README.md

```diff
@@ -43,6 +43,36 @@ python nutrition_labels/grant_tagger.py --training_data_file data/processed/trai
 
 where `vectorizer_type` can be 'count', 'tfidf' or 'bert', `model_type` can be 'naive_bayes', 'SVM' and 'log_reg', and `bert_type` (if using bert) can be 'bert' and 'scibert'.
 
+## finding_most_common_cohort.py / finding_most_common_references.py 
+
+To find the most common health data sets first run:
+
+```
+python nutrition_labels/finding_most_common_cohort.py
+```
+
+This uses data extracted from EPMC which has all the citations from all the papers produced from tech grants. 
+
+This then produces a list of most cited papers which have been manually filtered to find only the health data set papers. 
```

Comment:
Is the manual filtering the bit where you only include ones where the title has `data_set_terms = ['cohort','profile','resource','biobank','Cohort','Profile','Biobank','Resource']` in? If so, it'd be good to explain this here.

Reply:
Also, you could say: 
"this is outputted in 'data/processed/health_data_sets_list.csv'"